DatadogRUM/Tests/RUMMonitor/Scopes/RUMSessionScopeTests.swift

```diff
@@ -557,60 +588,6 @@ class RUMSessionScopeTests: XCTestCase {
         XCTAssertFalse(result)
     }
 
-    func testWhenScopeEnded_itSendsViewEventMessages() throws {
-        let featureScope = FeatureScopeMock()
-
-        // Given
-        let scope: RUMSessionScope = .mockWith(
-            parent: parent,
-            startTime: Date(),
-            dependencies: .mockWith(featureScope: featureScope)
-        )
-
-        let command = RUMStartViewCommand.mockWith(time: Date(), identity: .mockViewIdentifier(), name: "ActiveView")
-        // When
-        _ = scope.process(command: command, context: context, writer: writer)
-        // Then
-        let viewEventBaggage: RUMViewEvent = try XCTUnwrap(
-            featureScope.messagesSent().lastBaggage(withKey: RUMBaggageKeys.viewEvent)?.decode()
-        )
-        XCTAssertEqual(viewEventBaggage.view.name, "ActiveView", "It must send 'view event' message")
-
-        // When
-        _ = scope.process(command: RUMStopSessionCommand.mockWith(time: Date()), context: context, writer: writer)
-
-        // Then
-        let viewResetBaggage: Bool = try XCTUnwrap(
-            featureScope.messagesSent().lastBaggage(withKey: RUMBaggageKeys.viewReset)?.decode()
-        )
-        XCTAssertTrue(viewResetBaggage, "It must send 'view reset' message")
-    }
-
-    func testWhenScopeEnded_itDoesNotSendViewResetMessage() {
-        let featureScope = FeatureScopeMock()
-
-        // Given
-        let scope: RUMSessionScope = .mockWith(
-            parent: parent,
-            startTime: Date(),
-            dependencies: .mockWith(featureScope: featureScope)
-        )
-
-        let startViewCommand = RUMStartViewCommand.mockWith(time: Date(), identity: .mockViewIdentifier())
-        _ = scope.process(command: startViewCommand, context: context, writer: writer)
-        let startResourceCommand = RUMStartResourceCommand.mockWith(time: Date())
-        _ = scope.process(command: startResourceCommand, context: context, writer: writer)
-
-        // When
-        _ = scope.process(command: RUMStopSessionCommand.mockWith(time: Date()), context: context, writer: writer)
-        let stopResourceCommand = RUMStopResourceCommand.mockWith(resourceKey: startResourceCommand.resourceKey, time: Date())
-        _ = scope.process(command: stopResourceCommand, context: context, writer: writer)
-
-        // Then
-        let viewResetMessages = featureScope.messagesSent().filter { $0.asBaggage?.key == RUMBaggageKeys.viewReset }
-        XCTAssertEqual(viewResetMessages.count, 1, "It must send only one 'view reset' message")
-    }
```

Comment:
💡 I remove this test as it covers non-existing behaviour. Once `RUMStopSessionCommand` is received, the RUM monitor deallocates its scope, so it can no longer receive future commands (`RUMStopResourceCommand` in this case).